src/operator/tensor/elemwise_binary_op_basic.cu

```diff
@@ -3,45 +3,46 @@
  * \file elemwise_binary_scalar_op.cu
  * \brief GPU Implementation of unary function.
  */
-#include "./elemwise_unary_op.h"
 #include "./elemwise_binary_op.h"
 
 namespace mxnet {
 namespace op {
-NNVM_REGISTER_OP(elemwise_add)
-.set_attr<FCompute>("FCompute<gpu>", BinaryComputeWithHalf2<gpu, mshadow::op::plus>)
-.set_attr<FComputeEx>(FCOMP_EX_GPU, BinaryComputeEx<gpu, mshadow::op::plus>);
+MXNET_OPERATOR_REGISTER_BINARY_LAUNCH_CUDA(elemwise_add, gpu, mshadow::op::plus);
 
```

Comment:
Did you import the test in test_operator_gpu.py and verify that all these operator actually works on gpu? I'm sure the elemwise_add I implemented is cpu-only, because aux_data resides on gpu and cannot be accessed directly without being copied back to cpu